robocorp-code/vscode-client/src/inspector.ts

```diff
@@ -0,0 +1,198 @@
+import * as path from 'path';
+import {commands, Progress, ProgressLocation, window} from 'vscode'
+import {ROBOCORP_GET_LANGUAGE_SERVER_PYTHON_INFO} from './robocorpCommands';
+import {getRccLocation} from './rcc'
+import {getExtensionRelativeFile, verifyFileExists} from './files'
+import {listAndAskRobotSelection} from './activities'
+import {getSelectedLocator, getSelectedRobot, LocatorEntry} from './viewsCommon';
+import {sleep, Timing} from './time'
+import {execFilePromise, ExecFileReturn} from './subprocess'
+import * as roboConfig from './robocorpSettings';
+import {OUTPUT_CHANNEL} from './channel';
+
+interface InterpreterInfo {
+    pythonExe: string;
+    environ?: { [key: string]: string };
+    additionalPythonpathEntries: string[];
+}
+
+interface InterpreterInfo {
+    pythonExe: string;
+    environ?: { [key: string]: string };
+    additionalPythonpathEntries: string[];
+}
+
+let _cachedInspectorPythonInfo: InterpreterInfo;
+
+export async function openRobocorpInspector(locatorType?: string, locator?: LocatorEntry): Promise<void> {
+    let locatorJson;
+    const args: string[] = [];
+    let robot: LocalRobotMetadataInfo | undefined = getSelectedRobot('Please select a robot first.')?.robot;
+    if (!robot) {
+        // Ask for the robot to be used and then show dialog with the options.
+        robot = await listAndAskRobotSelection(
+          'Please select the Robot where the locators should be saved.',
+          'Unable to create locator (no Robot detected in the Workspace).'
+        );
+        if (!robot) return;
+    }
+    const robotYaml = robot.filePath;
+    locatorJson = path.join(robot.directory, "locators.json")
+    locatorJson = verifyFileExists(locatorJson) ? locatorJson : undefined;
+    let inspectorLaunchInfo = await commands.executeCommand(ROBOCORP_GET_LANGUAGE_SERVER_PYTHON_INFO) as InterpreterInfo | undefined;
```

Comment:
The actions are good when it's called from the language server or from another extension, but in this case I think we could call `getLanguageServerPythonInfo` directly (that way we don't need to cast anything).

Reply:
For this I was thinking that is there a possibility for cyclic dependencies and decided to go with the commands.executeCommand()-route, but it makes sense to use internals inside one extension

Reply:
changed to use the internal getLanguageServerPythonInfo()